web/pdf_presentation_mode.js

```diff
@@ -119,17 +119,34 @@ var PDFPresentationMode = (function PDFPresentationModeClosure() {
       return true;
     },
 
-    /**
-     * Switches page when the user scrolls (using a scroll wheel or a touchpad)
-     * with large enough motion, to prevent accidental page switches.
-     * @param {number} delta - The delta value from the mouse event.
-     */
-    mouseScroll: function PDFPresentationMode_mouseScroll(delta) {
+    _mouseWheel: function PDFPresentationMode_mouseWheel(evt) {
```

Comment:
Can we add a `@private` JSDoc comment above this method like the other private methods in this file?